frontend/src/components/ModalContent.vue

```diff
@@ -125,59 +122,83 @@ a {
   text-transform: none;
 }
 
-.buttons-list {
-  margin: 0.5rem 2rem;
+h1 {
+  font-family: $font-header;
+  font-size: 2.5rem;
+  color: $white;
+  word-break: break-word;
+  margin-right: auto;
+  @media only screen and (max-width: $phone) {
+    font-size: 2rem;
+    text-align: center;
+    margin-right: 0;
+  }
 }
 
-.text-list {
-  margin: 0.25rem 3rem;
+h2 {
+  font-family: $font-content;
+  font-size: 1.5rem;
+  color: black;
```

Comment:
use variable

Reply:
Well... we didn't have global black defined :D I'll work on that => promise (I realize that promise isn't usua;;y what you return in a function => just an inside joke :P )

Reply:
**Final changes:**
That's done